src/apolo_app_types/protocols/llm.py

```diff
@@ -125,9 +131,42 @@ class LLMApiKey(AbstractAppFieldType):
 
 
 class VLLMOutputsV2(AppOutputs):
-    chat_internal_api: OpenAICompatChatAPI | None = None
-    chat_external_api: OpenAICompatChatAPI | None = None
-    embeddings_internal_api: OpenAICompatEmbeddingsAPI | None = None
-    embeddings_external_api: OpenAICompatEmbeddingsAPI | None = None
-    llm: LLMModel | None = None
-    llm_api_key: LLMApiKey | None = None
+    chat_internal_api: OpenAICompatChatAPI | None = Field(
+        default=None,
+        json_schema_extra=SchemaExtraMetadata(
+            title="Chat Internal API", description="Chat Internal API "
+        ).as_json_schema_extra(),
+    )
+    chat_external_api: OpenAICompatChatAPI | None = Field(
+        default=None,
+        json_schema_extra=SchemaExtraMetadata(
+            title="Chat External API",
+            description="Chat External API description",
+        ).as_json_schema_extra(),
+    )
+    embeddings_internal_api: OpenAICompatEmbeddingsAPI | None = Field(
+        default=None,
+        json_schema_extra=SchemaExtraMetadata(
+            title="Embeddings Internal API",
+            description="Embeddings Internal API description",
+        ).as_json_schema_extra(),
+    )
+    embeddings_external_api: OpenAICompatEmbeddingsAPI | None = Field(
+        default=None,
+        json_schema_extra=SchemaExtraMetadata(
+            title="Embeddings External API", description="Embeddings External API"
+        ).as_json_schema_extra(),
+    )
+    llm: LLMModel | None = Field(
```

Comment:
Just to confirm, we are doing this, because `LLMModel`'s title and description are not displayed on the selector, right?

I mean, if we'd have `oneOf` three options, we'd have here title and description for all of them, but for `oneOf` case with only 2 options, one of which None, we need to propagate Title&Description for the underlying type, right?

Reply:
yes, exactly, unfortunately, we have to do this